pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1706,7 +1707,13 @@ class Timedelta(_Timedelta):
                 value = parse_timedelta_string(value)
             value = np.timedelta64(value)
         elif PyDelta_Check(value):
-            value = convert_to_timedelta64(value, 'ns')
+            # pytimedelta object -> microsecond resolution
```

Comment:
When the non-nano resolution docs are written, might be good to mention that `datetime.timedeltas` now default to microsecond resolution?